scriptmodules/system.sh

```diff
@@ -87,6 +87,12 @@ function get_os_version() {
                 __platform_flags+=" xbian"
             fi
             ;;
+         Devuan)
+            # Devuan 1 === Debian 8 Jessie, with the only modification
+            # of a different init system (it gets rid of systemD)
+            # so any Devuan version meets the requeriments
+            foo=bar
```

Comment:
is this here as a "noop" ? This PR needs a little work - what is `__os_debian_ver` set to in this case ?